test/unit/jest.conf.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
 		'^.+\\.(gql|graphql)$': 'jest-transform-graphql',
 		'^.+\\.vue$': 'vue-jest',
 		'^.+\\.js$': 'babel-jest',
+		'^.+\\.svg$': '<rootDir>/test/unit/specs/svgTransform.js',
```

Comment:
@emuvente Needed to add in this transform [to make Jest happy](https://vue-svg-loader.js.org/faq.html#how-to-use-this-loader-with-jest ). Wasn't sure where to put this svgTransform.js file though. Thoughts?

Reply:
I think making a new folder `/test/unit/transforms/` and putting it there would make sense